src/Analysis/Ast/Impl/Diagnostics/ErrorCodes.cs

```diff
@@ -25,5 +25,6 @@ public static class ErrorCodes {
         public const string UndefinedVariable = "undefined-variable";
         public const string VariableNotDefinedGlobally= "variable-not-defined-globally";
         public const string VariableNotDefinedNonLocal = "variable-not-defined-nonlocal";
+        public const string TypeVarLint = "typevar-linter";
```

Comment:
Something more descriptive since this is what user would use to enable/disable the diagnostic. `typevar-arguments`

Reply:
Done!